test/functional/wallet_mnemonicbits.py

```
@@ -17,14 +17,65 @@ def set_test_params(self):
+        for desc in descriptors:
+            if desc['desc'][:4] == 'pkh(':
+                pass
+                if mnemonic is None:
+                    mnemonic = desc['mnemonic']
+                else:
+                    assert_equal(mnemonic, desc['mnemonic'])
+            elif desc['desc'][:6] == 'combo(':
+                assert 'mnemonic' not in desc
+            else:
+                self.log.info(f"unknown type of descriptor: {desc['desc']}")
+                assert False
```

Ruff (0.8.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)